tests/001-handshake.cpp

```diff
@@ -50,7 +50,7 @@ namespace oxen::quic::test
             auto ep_notls = test_net.endpoint(default_addr);
             auto ep_tls = test_net.endpoint(local_addr);
 
-            REQUIRE_THROWS(ep_notls->listen());
+            // REQUIRE_THROWS(ep_notls->listen());  // Shouldn't compile if uncommented!
```

Comment:
This change is definitely an improvement. Small detail if you like: dropping `REQUIRE_THROWS` and just leaving `ep_notls->listen()` commented out would be marginally clearer

Reply:
agreed; that hadn't even occurred to me.